parametric/utils/test_agent.py

```diff
@@ -0,0 +1,4 @@
+def get_span(test_agent):
```

Comment:
@zacharycmontoya can we add this to the test_agent fixture (`_TestAgentAPI`) for convenience?

I'm ok holding off for now and refactoring it afterwards since this is just a nice-to-have.